src/Tvheadend.cpp

```diff
@@ -598,6 +602,43 @@ PVR_ERROR CTvheadend::RenameRecording ( const PVR_RECORDING &rec )
   return SendDvrUpdate(m);
 }
 
+PVR_ERROR CTvheadend::SetPlayCount ( const PVR_RECORDING &rec, int playcount )
```

Comment:
You use `playCount` in another place, I suggest to use it here too for consistency